src/test/resources/alma-fix/990103899140206441.json

```diff
@@ -3,6 +3,7 @@
   "almaMmsId" : "990103899140206441",
   "oclcNumber" : [ "1367753905" ],
   "zdbId" : "590016-5",
+  "dnbId" : "01335941X",
```

Comment:
This field `dnbId` has no entry in the `context.jsonld`.
Can you provide one ?

Reply:
I added it now